blocks/carousel/carousel.css

```diff
@@ -37,22 +37,33 @@
 }
 
 .carousel .carousel-slide > div {
+    flex-direction: column;
     font-family: var(--serif-font);
     font-weight: var(--font-weight-light);
     line-height: var(--line-height-120);
     letter-spacing: var(--letter-spacing-1-75);
     font-size: var(--font-size-28);
 }
 
-.carousel .carousel-slide > figure {
+/* Testimonial */
+.carousel.testimonial p {
+    margin-left: 1em;
+}
+
+.carousel .carousel-slide div {
+    display: flex;
+    flex-direction: column;
+}
+
+.carousel .carousel-slide div figure {
     align-content: center;
     align-items: center;
-    display: flex;
     flex-flow: column nowrap;
     gap: var(--spacer-layout-01);
     justify-content: space-between;
     margin: var(--spacer-layout-03) auto 0;
     width: fit-content;
+    flex-direction: row;
```

Comment:
put this line under a new grouping:

.carousel.testimonial .carousel-slide div figure {
    flex-direction: row;
    }